xbmc/platform/android/network/NetworkAndroid.cpp

```diff
@@ -360,14 +360,21 @@ void CNetworkAndroid::RetrieveInterfaces()
 
 void CNetworkAndroid::onAvailable(const CJNINetwork n)
 {
-  CLog::Log(LOGDEBUG, "CNetworkAndroid::onAvailable The default network is now: {}", n.toString());
+  CLog::Log(LOGDEBUG, "CNetworkAndroid::onAvailable: The default network is now: {}", n.toString());
 
   CJNIConnectivityManager connman{CJNIContext::getSystemService(CJNIContext::CONNECTIVITY_SERVICE)};
   CJNILinkProperties lp = connman.getLinkProperties(n);
 
   if (lp)
   {
     CJNINetworkInterface intf = CJNINetworkInterface::getByName(lp.getInterfaceName());
+    if (xbmc_jnienv()->ExceptionCheck())
+    {
+      xbmc_jnienv()->ExceptionClear();
+      CLog::Log(LOGERROR, "CNetworkAndroid::onAvailable: Cannot get interface by name: {}",
+                lp.getInterfaceName());
```

Comment:
wont lp.getInterfaceName always be null in this situation? what value does logging it give?

Reply:
This code captures the exceptions that can be thrown when invoking the `getByName()` method, one can be `NullPointerException` when receiving a parameter that is `null`, but they can be others such as `SocketException` and in this case it would have value.